### PR TITLE
Docs: Mc: fix version in setup guide

### PR DESCRIPTION
**Pull request by Joethepic**
This makes it so it tells you to load 1.19 instead of 1.18.2.

### CODE REVIEW

**Comment by KonoTyran** [comment]
I thought I removed all references to version numbers to avoid this problem. (the version you need to load depends on what the seed was generate as so it will never be 100% accurate. Please just remove the version number completely instead of changing it.

**Comment by KonoTyran** [comment]
@Joethepic You have meged in changes numerous times without addressing the issues I pointed out? Did you miss where I pointed them out?

**Comment by Joethepic** [comment]
I must of or GitHub glitched because the only comment I see from you is telling me to remove the version completely instead of just changing it
0B7943D3-2C13-4F91-88D2-F60761422E31

**Comment by black-sliver** [comment]
@Joethepic maybe i wasn't clear in the previous PR. You should **not** merge updates from the base branch to your PRs unless there is a reason to. Doing so will generate empty commits that don't change anything but still send emails to everyone subscribed to the PR.

Let me try to explain why this is: when you PR something you send a diff between two points in your vs. base history. When you "merge branch 'main' into ..." you forward both points by the same stuff, resulting in an empty change to the PR, but it still updates the history, so you produce a merge commit that gets emailed.

File: worlds/minecraft/docs/minecraft_en.md [hunk 1]
```diff
@@ -33,12 +33,12 @@ leave this window open as this is your server console.
 
 ### Connect to the MultiServer
 
-Using Minecraft 1.18.2 connect to the server `localhost`.
+Open Minecraft go to multiplayer option and then connect to the server `localhost`.
```
Comment (SoldierofOrder) [inline worlds/minecraft/docs/minecraft_en.md:36]:
```suggestion
Open Minecraft, go to `Multiplayer > Direct Connection`, and join the `localhost` server address.
```
Suggesting an update to use the exact language Minecraft uses.

File: worlds/minecraft/docs/minecraft_en.md [hunk 2]
```diff
@@ -33,12 +33,12 @@ leave this window open as this is your server console.
...
 
 If you are using the website to host the game then it should auto-connect to the AP server without the need to `/connect`
 
 otherwise once you are in game type `/connect <AP-Address> (Port) (Password)` where `<AP-Address>` is the address of the
-Archipelago server. `(Port)` is only required if the Archipelago server is not using the default port of 38281.
+Archipelago server. `(Port)` is only required if the Archipelago server is not using the default port of 38281. In order to type in the port use a space instead of a colon
```
Comment (SoldierofOrder) [inline worlds/minecraft/docs/minecraft_en.md:41]:
```suggestion
Archipelago server. `(Port)` is only required if the Archipelago server is not using the default port of 38281. Note that there is no colon between `<AP-Address>` and `(Port)`.
```
Makes the language clearer to a first-time AP user, without sacrificing meaning.

**Review by SoldierofOrder — APPROVED** [review]

**Review by KonoTyran — APPROVED** [review]

**Pull request merged**